CONTRIBUTING.md

```diff
@@ -160,7 +160,7 @@ main();
 #### C#
 
 ```csharp
-using static System.Console;
+using System.Console;
```

Comment:
@aashutoshrathi you can't use `System.Console` without `static` keyword, so make it `using System;`.